### ISSUE TITLE
Plugins/Grafana OnCall configuration page endless loading with 502 error 

### ISSUE THREAD

**Issue by Gufderald** [grafana/oncall#622]
Hi!

Faced with next issue while trying to configure OnCall plugin:
![image](https://user-images.githubusercontent.com/75615922/194401739-fa579b54-2953-4054-87f0-c6454fd3d2f9.png)

I can see next errors in Grafana logs:
```
logger=data-proxy-log userId=1 orgId=1 uname=admin app=grafana-oncall-app path=/api/plugin-proxy/grafana-oncall-app/api/internal/v1/plugin/sync remote_addr=172.16.0.246 referer=http://172.16.0.77:3000/plugins/grafana-oncall-app t=2022-10-06T19:22:20.14156262Z level=error msg="Proxy request failed" err="unsupported protocol scheme \"\""
server.go:3160: http: superfluous response.WriteHeader call from github.com/grafana/grafana/pkg/web.(*responseWriter).WriteHeader (response_writer.go:67)
logger=context traceID=00000000000000000000000000000000 userId=1 orgId=1 uname=admin t=2022-10-06T19:22:20.142752347Z level=error msg="Request Completed" method=POST path=/api/plugin-proxy/grafana-oncall-app/api/internal/v1/plugin/sync status=502 remote_addr=172.16.0.246 time_ms=6 duration=6.557124ms size=91 referer=http://172.16.0.77:3000/plugins/grafana-oncall-app traceID=00000000000000000000000000000000
```

Running hobby environment from docker-compose.yml. Tried Grafana images v8.5.3, v9.0.0-beta3, v9.1.7, with different values for DOMAIN(http and https), with different docker and docker-compose versions; all reproduces. 

Please let me know if I can provide some more information.

**Comment by vadimkerr** [comment]
Thank you for the bug report!
This issue should be fixed by [this commit](https://github.com/grafana/oncall/commit/a96f7315d46c4f883500e8d5bd3990115bfbe753) and released in version [v1.0.40](https://github.com/grafana/oncall/releases/tag/v1.0.40), so please try [updating to the latest version](https://github.com/grafana/oncall#update-version) and see if the issue is resolved.

**Comment by Gufderald** [comment]
Thank you, now it works as expected! Closing.
